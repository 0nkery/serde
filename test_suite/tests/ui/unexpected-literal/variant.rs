#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
enum E {
    #[serde("literal")]
    V,
}

fn main() {}
