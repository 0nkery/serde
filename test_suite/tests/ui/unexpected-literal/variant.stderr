error: unexpected literal in serde variant attribute
 --> $DIR/variant.rs:6:13
  |
6 |     #[serde("literal")]
  |             ^^^^^^^^^

error: aborting due to previous error

